photo-gallery.js

```diff
@@ -44,8 +57,12 @@ class FlitciePhotoGallery extends LitElement {
             return html`
               <a href="${actualPath}" on-click=${event => this.goToUrl(event, actualPath)}>
                 <h3>${title}</h3>
-                <img src="https://flitcie.ch.tudelft.nl/var/thumbs${imagePath}">
-              </a>
+                <div>
+                  <div>
```

Comment:
Can we do this without adding these extra divs?

Reply:
I don't think so, since the behavior of the image in the div changes when the div is present.